packages/atlas/src/components/_playlist/PlaylistListItem/PlaylistListItem.tsx

```diff
@@ -0,0 +1,120 @@
+import { useState } from 'react'
+import { CSSTransition } from 'react-transition-group'
+
+import { SvgActionEdit, SvgActionHide, SvgActionLinkUrl, SvgActionPlay, SvgActionShow } from '@/assets/icons'
+import { Text } from '@/components/Text'
+import { Tooltip } from '@/components/Tooltip'
+import { Button } from '@/components/_buttons/Button'
+import { ColumnGapBlock, RowGapBlock } from '@/components/_layouts'
+import { VideoThumbnail } from '@/components/_video/VideoThumbnail'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { cVar } from '@/styles'
+
+import {
+  Cell,
+  HoverContainer,
+  PlaylistDescription,
+  StyledPill,
+  StyledSvgActionTrash,
+  Wrapper,
+} from './PlaylistListItem.styles'
+
+export const PlaylistListItem = () => {
+  const mdMatch = useMediaMatch('md')
+  const [isHovered, setIsHovered] = useState(false)
+
+  const actionButtons = (
+    <ColumnGapBlock gap={1}>
+      <Tooltip text="Edit playlist" placement="top">
+        <Button icon={<SvgActionEdit />} variant="tertiary" />
+      </Tooltip>
+      <Tooltip text="Cody link" placement="top">
+        <Button icon={<SvgActionLinkUrl />} variant="tertiary" />
+      </Tooltip>
+      <Tooltip text="Play in Joystream" placement="top">
+        <Button icon={<SvgActionPlay />} variant="tertiary" />
+      </Tooltip>
+      <Tooltip text="Delete playlist" placement="top">
+        <Button icon={<StyledSvgActionTrash />} variant="tertiary" />
+      </Tooltip>
+    </ColumnGapBlock>
+  )
+
+  const visibilityPillProps = isHovered
+    ? { label: 'Public', icon: <SvgActionShow /> }
+    : { label: 'Unlisted', icon: <SvgActionHide /> }
+
+  return (
+    <Wrapper onMouseEnter={() => setIsHovered(true)} onMouseLeave={() => setIsHovered(false)}>
+      <ColumnGapBlock gap={4} align="center">
+        <VideoThumbnail
```

Comment:
Thumbnails are smaller than in the design. Could we improve that? 
![image](https://user-images.githubusercontent.com/51168865/212832442-875737f8-c491-4b94-8dba-e9a036b9474a.png)